internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/kafka/consumer/AdminConsumptionTaskIntegrationTest.java

```diff
@@ -243,6 +272,73 @@ public void testUpdateAdminOperationVersion() {
     }
   }
 
+  @Test(timeOut = 2 * TIMEOUT)
+  public void testAdminConsumptionTaskWithSpecificWriterId() throws IOException {
+    int adminConsumptionMaxWorkerPoolSize = 3;
+    // Use a specific version to test the serialization and deserialization of admin operation.
+    int writerSchemaId = AdminOperationSerializer.LATEST_SCHEMA_ID_FOR_ADMIN_OPERATION - 10;
```

Comment:
Let's change it to `AdminOperationSerializer.LATEST_SCHEMA_ID_FOR_ADMIN_OPERATION - 1` please, so that when dev adds a new version, we will test it against the previous latest schema directly.

Reply:
Changed to verify for the latest 5 versions.